readthedocs/core/static-src/core/js/doc-embed/rtd-data.js

```diff
@@ -24,14 +29,15 @@ var configMethods = {
     },
 
     get_theme_name: function () {
-        // Crappy heuristic, but people change the theme name on us.  So we have to
-        // do some duck typing.
-        if (this.theme !== constants.THEME_RTD) {
+        // Crappy heuristic, but people change the theme name on us.
+        // So we have to do some duck typing.
+        if (this.theme !== constants.THEME_RTD && this.theme !== constants.THEME_ALABASTER) {
             if (this.theme === constants.THEME_MKDOCS_RTD) {
                 return constants.THEME_RTD;
-            }
-            if ($('div.rst-other-versions').length === 1) {
+            } else if ($('div.rst-other-versions').length === 1) {
                 return constants.THEME_RTD;
+            } else if ($('div.sphinxsidebar > div.sphinxsidebarwrapper').length === 1) {
+                return constants.THEME_ALABASTER;
```

Comment:
Is this unique to alabaster? It feels like something that would be on almost all themes, as it's inherited from the Sphinx `basic` theme: https://github.com/sphinx-doc/sphinx/blob/master/sphinx/themes/basic/layout.html#L52-L53

Reply:
Ok. Maybe I need to rethink this. That's a good point.

Reply:
We are using the same selector to inject the Ads in this line https://github.com/rtfd/readthedocs.org/pull/4499/files#diff-549a0ee75d29b10f27785c6d36e8ce2aR28

So, if the "semantic meaning" of that element from the base theme is what we are selecting, it may makes sense on other themes also.

It would be good to test other themes, though.

Reply:
Yea, it's a common element across most themes with sidebars. Doing that hueristic selector would definitely let us put ads on more themes, but we should just know that's what we're doing :)

Reply:
My immediate goal is to get ads on the pallets themes (which all have different names). Maybe I'll add some constants for "alabaster-like" theme names. From there if we want to iterate and put ads on more themes we can do that in the future.